### PR TITLE
Disabled layers with text-name fixes #7594

### LINKED ISSUE #7594: [Wizards migration] any style migration is "enabling" labels
Looks like `labels -> enabled` is set as true always, no matter if labels was enabled in the old editor or not.

### PR DESCRIPTION
**Pull request by juanignaciosl**
This fixes #7594

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@javitonino CR this, please.

File: app/controllers/carto/api/layer_presenter.rb [hunk 1]
```diff
@@ -673,12 +673,12 @@ def generate_labels(wpp)
...
 
         apply_direct_mapping(labels, wpp, TEXT_DIRECT_MAPPING)
-        labels['attribute'] = nil if labels['attribute'] == 'None'
+        labels['attribute'] = nil if labels['attribute'].to_s.downcase == 'none'
 
         merge_into_if_present(labels, 'fill', generate_labels_fill(wpp))
         merge_into_if_present(labels, 'halo', generate_labels_halo(wpp))
 
-        labels['enabled'] = true unless labels.empty?
+        labels['enabled'] = labels['attribute'].present?
```
Comment (javitonino) [inline app/controllers/carto/api/layer_presenter.rb]: This sets enabled to false if the `text-name` is empty. Is this desired? If not, maybe use `!labels['attribute'].nil?`.

Comment (javitonino) [inline app/controllers/carto/api/layer_presenter.rb]: Nevermind, this can never be blank. I thought it was user input, but it is a column name.

**Comment by javitonino** [comment]
:shipit: Don't mind the comments.

**Pull request merged**